README.md

```diff
@@ -87,4 +95,8 @@ The most common mood is :-)
 
 **think**: Should you use an array or a hash to solve this problem? Why?
 
+I used a hash of arrays because I wanted to access collected values by total, and this way I can call a key to see which one holds the most values.
```

Comment:
Hmm... actually. Do you think it's important that you push each of the emoticons into arrays? Or do you think you could have just used a fixnum to count. I'll argue that a fixnum would be sufficient because 1) it uses less space and 2) having arrays filled with emoticons doesn't really gain us anything.